src/Component/Page/Home.purs

```diff
@@ -0,0 +1,208 @@
+  = FeedTab
+  | GlobalTab
+  | TagTab String
+
+data Query a
+  = Init a
+  | LoadTags a
+  | Navigate Route a
+  | ShowTab Tab Event a
+  | LoadArticles ArticleParams a
+
+component
+  :: forall m
+   . MonadAff m
+  => MonadAsk Env m
+  => Now m
+  => LogMessages m
+  => ManageResource m
+  => Navigate m
+  => H.Component HH.HTML Query Unit Void m
+component =
+  H.lifecycleParentComponent
+    { initialState: const { tags: NotAsked, articles: NotAsked, tab: GlobalTab } 
+    , render
+    , eval
+    , receiver: const Nothing
+    , initializer: Just $ Init unit
+    , finalizer: Nothing
+    }
+
+  where 
+
+  eval :: Query ~> H.ParentDSL State Query (Const Void) Void Void m
+  eval = case _ of
+    Init a -> do
+      handle <- H.fork $ eval $ LoadTags a
+      eval $ LoadArticles noArticleParams a
+    LoadTags a -> do
+      H.modify_ _ { tags = Loading}
+      tags <- H.lift getTags
+      H.modify_ _ { tags = either Failure Success tags }
+      pure a
+    LoadArticles params a -> do
+      H.modify_ _ { articles = Loading }
+      articles <- H.lift $ getArticles params
+      H.modify_ _ { articles = either Failure Success articles }
+      pure a      
+    Navigate route a -> do
+      navigate route
+      pure a
+    ShowTab tab ev a -> do
+      H.liftEffect $ preventDefault ev
+      H.modify_ _{ tab = tab }
+      case tab of 
+        FeedTab -> pure a -- TODO
+        GlobalTab -> eval $ LoadArticles noArticleParams a
+        TagTab tag -> eval $ LoadArticles noArticleParams{ tag = Just tag } a
+  
+
+  render :: State -> H.ParentHTML Query (Const Void) Void m
+  render state@{ tags, articles} =
+    HH.div_
+    [ header Home Navigate
+    , HH.div
+      [ HP.class_ CC.homePage ]
+      [ banner
+      , HH.div
+        [ HP.classes [ HB.container, CC.page ] ]
+        [ HH.div
+          [ HP.class_ HB.row ]
+          [ mainView state
+          , HH.div
+            [ HP.class_ HB.colMd3 ]
+            [ HH.div
+              [ HP.class_ CC.sidebar ]
+              [ HH.p_ [ HH.text "Popular Tags" ]
+              , renderTags tags
+              ]
+            ]
+          ]
+        ]
+      ]
+    , footer
+    ]
+
+  banner :: forall p i. HH.HTML p i
+  banner =
+    HH.div
+    [ HP.class_ CC.banner ]
+    [ HH.div
+      [ HP.class_ HB.container ]
+      [ HH.h1
+        [ HP.class_ CC.logoFont ]
+        [ HH.text "conduit" ]
+      , HH.p_ [ HH.text "A place to share your knowledge." ]
+      ]
+    ]
+
+  mainView :: forall p. State -> H.HTML p Query
+  mainView state =
+    HH.div
+    [ HP.class_ HB.colMd9 ]
+    [ HH.div
+      [ HP.class_ CC.feedToggle ]
+      [ HH.ul
+        [ HP.classes [ HB.nav, HB.navPills, CC.outlineActive ] ]
+        [ yourFeedTab state.tab
+        , globalFeedTab state.tab
+        , tagFilterTab state.tab
+        ]
+      ]
+    , articleList state.articles
+    ]
+
+  yourFeedTab :: forall p i. Tab -> HH.HTML p i
+  yourFeedTab tab =
+    let modifiers = case tab of
```

Comment:
This creates some funky indentation. Mind moving this to a `where` clause instead?